packages/filesystem/src/common/filesystem.ts

```diff
@@ -37,6 +37,11 @@ export interface FileSystem extends JsonRpcServer<FileSystemClient> {
      */
     setContent(file: FileStat, content: string, options?: { encoding?: string }): Promise<FileStat>;
 
+    /**
+     * Updates the content in base64 replacing its previous value.
+     */
+    setContentBase64(file: FileStat, content: string): Promise<FileStat>;
```

Comment:
Can we do it via `setContents`?  maybe add some new options

cc @svenefftinge

Reply:
It seems that you don't need it, try to call `setContent(file, content, { encoding: 'base64' });`

Reply:
Ah you're right, it works:

```
$ cat test.js 
fs = require('fs')

fs.writeFile('hello.txt', 'SGVsbG8sIHdvcmxkIQo===', 'base64', function (err) {
	console.log("Done", err);
});
$ node test.js 
Done null
$ cat hello.txt 
Hello, world!
```

I am a confused by what the encoding means here, I thought it was the encoding in which to encode the text in the file.  But when specifying 'base64' it means how to decode the data before saving it.  Anyway, good news that it works!

Reply:
OK, remove the setcontentBase64 and use the previous method: setContent